axelrod/moran.py

```diff
@@ -56,6 +56,8 @@ def __init__(
         interaction_graph: Graph = None,
         reproduction_graph: Graph = None,
         fitness_transformation: Callable = None,
+        mutation_method="transition",
```

Comment:
how do we feel about having booleans instead of passing a string argument?

```
atomic=False, transition=True
```

Reply:
You can't have both at the same time so I'd prefer not to use individual booleans. I considered using an enum instead.